src/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -350,6 +351,57 @@ public string GetString()
             return _parent.GetString(_idx, JsonTokenType.String);
         }
 
+        /// <summary>
+        ///   Attempts to represent the current JSON string as bytes assuming it is <see cref="Base64"/> encoded.
+        /// </summary>
+        /// <param name="value">Receives the value.</param>
+        /// <remarks>
+        ///  This method does not create a byte[] representation of values other than <see cref="Base64"/> encoded JSON strings.
+        /// </remarks>
+        /// <returns>
+        ///   <see langword="true"/> if the entire token value is encoded as valid <see cref="Base64"/> text and can be successfully decoded to <see cref="Base64"/> bytes.
+        ///   <see langword="false"/> otherwise.
+        /// </returns>
+        /// <exception cref="InvalidOperationException">
+        ///   This value's <see cref="Type"/> is not <see cref="JsonValueType.String"/>.
+        /// </exception>
+        /// <exception cref="ObjectDisposedException">
+        ///   The parent <see cref="JsonDocument"/> has been disposed.
+        /// </exception>
+        public bool TryGetBytes(out byte[] value)
+        {
+            CheckValidInstance();
+
+            return _parent.TryGetValue(_idx, out value);
+        }
+
+        /// <summary>
+        ///   Gets the value of the element as bytes.
+        /// </summary>
+        /// <remarks>
+        ///   This method does not create a byte[] representation of values other than <see cref="Base64"/> encoded JSON strings.
```

Comment:
Why would this Base64 be a see? It feels like it's talking about the concept of base64, not the specific type.

Reply:
Just as note: my suggestions are to make it consistent with this.

So if this see goes, ignore these suggestions.

Reply:
I had similar concern when I added the link. What should this be instead, just plain text?